lib/API/MP/v1/CustomFields.php

```diff
@@ -41,18 +49,14 @@ public function getSubscriberFields(): array {
       ],
     ];
 
-    $customFields = CustomField::selectMany(['id', 'name', 'type', 'params'])->findMany();
+    $customFields = $this->customFieldsRepository->findAll();
```

Comment:
Before we were getting from the database just the columns 'id', 'name', 'type', 'params', and now we are getting all the fields from this particular table. This probably is not a big deal in this case as this table is unlikely to be bigger. But, as a good practice, what do you think of using another method that would let us specify which fields to retrieve and leave `findAll()` only for the instances when we want all the fields?

Reply:
I would argue it is a good practice in the Doctrine to load the complete entities for readability and maintainability purposes. If the entity is only partial, it can someone catch a surprise. If we need to optimise we can but I wouldn't do it when not needed.

Reply:
That makes sense